dotcom-rendering/src/web/components/Lineup.tsx

```diff
@@ -76,31 +76,19 @@ export const Lineup = ({ players }: Props) => {
 						<div
 							css={css`
 								font-weight: bold;
+								width: 30px;
 							`}
 						>
 							{player.shirtNumber}
 						</div>
-						<div
-							css={css`
-								position: absolute;
-								left: 40px;
-							`}
-						>
-							<Row>
-								{player.name}
-								{player.events.map((event: EventType) => (
-									<div
-										css={css`
-											margin-left: 4px;
-										`}
-									>
-										<Event
-											type={event.eventType}
-											time={event.eventTime}
-										/>
-									</div>
-								))}
-							</Row>
+						<div>
```

Comment:
This feels simpler, was there a change in design that allowed us to achieve this or is the result equivalent?

Reply:
We needed to ensure it fit better within the context, and the old way was causing overflow issues.